engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/util/ExpectStringNode.java

```diff
@@ -31,6 +32,11 @@ String doString(String str) {
     return str;
   }
 
+  @Specialization
+  String doWarning(WithWarnings value) {
```

Comment:
Is this correct after introduction of `WarningsLibrary`? Shouldn't one guard `WarningsLibrary.hasWarnings` and then `WarningsLibrary.removeWarnings(value)`?